js/deer-config.js

```diff
@@ -689,4 +689,4 @@ function debounce(func,timeout = 500) {
         clearTimeout(timeRemains)
         timeRemains = setTimeout(()=>func.apply(this,args),timeRemains)
     }
-}
```

Comment:
Oops!  Not a big deal.  If you can, undo this change so there is no diff on this file.